Scripts/Mac/PrepFunctionalTests.sh

```diff
@@ -36,13 +36,6 @@ fi
 
 git-credential-manager install
 
-# If our Java version is 9+ (the formatting of 'java -version' changed in Java 9), work around
```

Comment:
Why did you remove all of this?

Reply:
No longer needed now that 2.0.4 of the GCM is rolled out with the real fix :)